CHANGELOG.md

```
@@ -1,1427 +1 @@
-- Init repo by @mchmarny
-- Replace file-existence-action with hashFiles by @mchmarny
-- Replace ko-build/setup-ko with go install by @mchmarny
-- Remove Homebrew and update org to NVIDIA by @mchmarny
-- Update settings by @mchmarny
-- Remove code owners for now by @mchmarny
-- Update project docs and setup by @mchmarny
-- Update contributing doc by @mchmarny
-- Remove badges not supported in local repos by @mchmarny
-
-<!-- Generated by git-cliff -->
+see https://github.com/NVIDIA/aicr/releases
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)